pages/spaces.jsx

```diff
@@ -114,7 +114,16 @@ var ExpereincesInfo = [
     name: "Artist Open Studio",
     description:
     (<div>
-      <p></p>
+      <p>Upholding a healthy internet requires diverse voices and approaches—like art and design.</p>
+      <p>Art and design play key roles in engaging new audiences around issues like privacy and inclusion, and in shaping critical thinking around a healthy web. To highlight these artistic perspectives, MozFest—in collaboration with Digital Learning at V&A and Tate—has invited artists to address each of the five thematic issue areas of the festival through an installation and on-site lab:</p>
+        <ul>
+          <li>An Xiao Mina (Web Literacy)</li>
+          <li>Paolo Cirio (Privacy and Security)</li>
+          <li>Archana Prasad (Decentralization)</li>
+          <li>Gretta Louw (Open Innovation)</li>
+          <li>Brooklyn Pakathi (Digital Inclusion)</li>
+        </ul>
+      <p>Participants will engage with the artists in an open lab setting. Artists will share their practices, and also develop collaborative work with the public.</p>
```

Comment:
Let's don't wrap `<ul>` in `<p>`.  

https://www.w3.org/TR/html401/struct/text.html#h-9.3.1
> The P element represents a paragraph. It cannot contain block-level elements (including P itself).